webgpu/lessons/webgpu-inter-stage-variables.md

```diff
@@ -308,18 +308,20 @@ And we'd have to update our pipeline creation to use these
 
 ```
 
-And this would also work
+These modifications render the same:
```

Comment:
I just used 'render' as a similar verb for 'work'; your reasoning definitely makes sense to me though.

Maybe suggest something like:

>These changes work the same:

Also could just skip any word change (I tend to be too wordy anyway) and just append the colon:

>- And this would also work
>+ And this would also work:

Reply:
[resolution commit](../pull/144/commits/45361680a6419ad33822d63aec48e56d6b53491b)